plugins/samples/overwrite_header/tests.textpb

```diff
@@ -1,37 +1,47 @@
 # Expect no matches, so no overwrite.
 test {
-  name: "NoOverwrite"
-  benchmark: true
+  name: "NoOverwriteRequestHeader"
```

Comment:
naming nit: You might call this DontAddRequestHeader for clarity

If you change the response path behavior as I suggested, you could call that one DoAddResponseHeader